swmm-toolkit/src/swmm/toolkit/solver.i

```diff
@@ -59,4 +322,69 @@ int  swmm_getMassBalErr(float *OUTPUT, float *OUTPUT, float *OUTPUT);
 int  swmm_close(void);
 int  swmm_getVersion(void);
 
+
+// TOOLKIT API 
+int  swmm_getAPIError(int ErrorCodeAPI, char **errorMsg);
```

Comment:
So which approach do you think is better. Repeating the function declarations here or annotating the header file found in the swmm-solver submodule? There is no right answer they both have advantages and disadvantages.

Reply:
I prefer function declaration within solver interface file. That way it is easy to identify if swig has been properly set up for the api function.

Reply:
I agree. Should I use underscore instead? Or should I use the same variable from SWMM even though it is not pythonic?

Reply:
I say use something that is pythonic.